apps/opencs/view/render/terraintexturemode.hpp

```diff
@@ -66,6 +79,9 @@ namespace CSVRender
             /// \brief Handle brush mechanics, maths regarding worldspace hit etc.
             void editTerrainTextureGrid (const WorldspaceHitResult& hit);
 
+            /// \brief Handle brush mechanics for texture selection
+            void selectTerrainTextures (std::pair<int, int>, unsigned char, bool);
```

Comment:
Since the coords pair isn't modified, it should be passed by reference, not a copy. The arguments should be named here for clarity of their purpose too.

Reply:
True, I'll fix this.